src/universal/subscriptions/TeamSubscription.js

```diff
@@ -8,37 +8,44 @@ import {killMeetingTeamUpdater} from 'universal/mutations/KillMeetingMutation';
 import {promoteFacilitatorTeamUpdater} from 'universal/mutations/PromoteFacilitatorMutation';
 import {removeTeamMemberTeamUpdater} from 'universal/mutations/RemoveTeamMemberMutation';
 import {requestFacilitatorTeamUpdater} from 'universal/mutations/RequestFacilitatorMutation';
-import {removeOrgUserTeamUpdater} from 'universal/mutations/RemoveOrgUserMutation';
+import {removeOrgUserTeamOnNext, removeOrgUserTeamUpdater} from 'universal/mutations/RemoveOrgUserMutation';
 import {startNewMeetingTeamOnNext} from 'universal/mutations/StartNewMeetingMutation';
+import {navigateMeetingTeamOnNext} from 'universal/mutations/NavigateMeetingMutation';
+import {promoteNewMeetingFacilitatorTeamOnNext} from 'universal/mutations/PromoteNewMeetingFacilitatorMutation';
 
 const subscription = graphql`
   subscription TeamSubscription {
     teamSubscription {
       __typename
-      ...AcceptTeamInviteEmailMutation_team
-      ...AcceptTeamInviteMutation_team
-      ...AddTeamMutation_team
-      ...AddTeamMutation_team
-      ...ArchiveTeamMutation_team,
-      ...EndMeetingMutation_team
-      ...KillMeetingMutation_team
-      ...MoveMeetingMutation_team
-      ...PromoteFacilitatorMutation_team
-      ...RemoveTeamMemberMutation_team
-      ...RemoveOrgUserMutation_team
-      ...RequestFacilitatorMutation_team
-      ...StartMeetingMutation_team
-      ...StartNewMeetingMutation_team
-      ...UpdateCheckInQuestionMutation_team
-      ...UpdateCreditCardMutation_team
-      ...UpdateTeamNameMutation_team
-      ...UpgradeToProMutation_organization
+      ...AcceptTeamInviteMutation_team @relay(mask: false)
```

Comment:
now that we use onNext instead of updaters, we need to turn off masking so we can get the whole payload